spec/descriptions/agentMonitoringEvents.md

```diff
@@ -1,11 +1,5 @@
 This endpoint retrieves all agent monitoring events for the requested timeframe.
+The timeframe is defined as [from, to] or [to - windowSize, to].
 
-### Query Parameters:
-
-- **to:** The end of the requested timeframe as a Unix timestamp. The current service time is used as a default if not provided. Please note that it is recommended to fetch historical events with a delay using a `to` timestamp in the past due to eventual consistency. It can take up to **2 minutes** in some cases until an event update is reflected in the results.
-- **windowSize:** The size of the requested timeframe in milliseconds relative to `to`. If neither `windowSize` nor `from` is provided, then a default windowSize of *10 minutes* is used.
-- **from:** As an alternative to defining the `windowSize`, the `from` query parameter can be used, which spans the timeframe in range `[from, to)`.
-- **excludeTriggeredBefore:** Whether to exclude events that have been triggered before the requested timeframe in order to enable searching for events that have started within the given timeframe, excluding events that are previously active already. This is useful for 3rd party integrations that fetch events from Instana with a scheduled batch job in a fixed interval using tumbling windows, when you only care about new events.
-This option is more restrictive than `filterEventUpdates` and does not inform about event state updates that got `CLOSED` in the timeframe of the query if not also the start time of the event is within that query timeframe.
-- **filterEventUpdates:** Filters results to event updates only. This means that an event is only included when its event state changed in the given query timeframe. This is useful for 3rd party integrations that fetch events from Instana with a scheduled batch job in a fixed interval using a tumbling windows, when you care about event state updates.
```

Comment:
These param descriptions are now removed, because they are now instead described in-pace via the `@Parameter` annotations? Is that correct?

Reply:
Yes, but in less detail. I think the extra information will be added to the upcoming guides that April plans to put together.

Reply:
> I'm pretty sure the extra information will be added to the upcoming guides that April plans to put together.

I'm not sure that I understand the motivation behind removing the detailed information about these parameter endpoints from the documentation source. If this context will be added elsewhere it seems like we should at least link to it from here?

Reply:
CC @aprilwebster

Reply:
@csaroff @b3nk4n - two things regarding descriptions
1. **Endpoint descriptions** -- Move to the `ui-backend`. We need to move the endpoint description (which is what these files are for, not parameter descriptions) into the ui-backend code annotations. When we move to the API Hub, we will need to fully generate the openapi.json from that codebase. Only rendering of the openapi.json should be handled in this repo.
2. **General info in endpoint descriptions** -- regarding general information about filtering, etc, that should go into the Getting Started docs. This is typically how it's handled in API documentation in industry. We don't want to overload the API endpoint description with this information.

Let's revisit some of the additional context/semantics that are included in the Events endpoint descriptions to figure out what home makes sense for it. Right now some of this context is a bit confusing and we should probably revisit it and provide extra examples? Perhaps a doc dedicated to Events in the Getting Started documentation makes most sense and we can link it from the endpoints.